node/cmd/guardiand/node.go

```diff
@@ -1721,6 +1724,10 @@ func runNode(cmd *cobra.Command, args []string) {
 			Commitment:    rpc.CommitmentFinalized,
 		}
 		watcherConfigs = append(watcherConfigs, wc)
+
+		if *solanaShimContract != "" {
+			featureFlags = append(featureFlags, "solshim")
```

Comment:
Would whoever is monitoring want to know the address of the shim that is set?

Reply:
Welp, you're the one who was worried about the length of the feature string, but I can add it. 😁